### PR TITLE
Update outdated content in dev-1.16-ko.1

### LINKED ISSUE #16438: Outdated files in the dev-1.16-ko.1 branch
# This is a Bug Report
## Problem
Outdated files in the dev-1.16-ko.1 branch.

### 17 files to be modified 
1. [x] content/en/_index.html 1 file changed, 1 insertion(+)
1. [x] content/en/docs/concepts/architecture/cloud-controller.md 1 file changed, 5 insertions(+), 4 deletions(-)
1. [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 11 insertions(+)
1. [x] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 48 insertions(+), 4 deletions(-)
1. [x] content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 21 insertions(+), 2 deletions(-)
1. [x] content/en/docs/reference/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/reference/glossary/applications.md 1 file changed, 1 deletion(-)
1. [x] content/en/docs/reference/glossary/static-pod.md 1 file changed, 8 insertions(+), 4 deletions(-)
1. [x] content/en/docs/reference/glossary/workload.md 1 file changed, 8 insertions(+), 14 deletions(-)
1. [x] content/en/docs/setup/_index.md 1 file changed, 2 insertions(+), 1 deletion(-)
1. [x] content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [x] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 4 insertions(+)
1. [x] content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md 1 file changed, 147 insertions(+), 100 deletions(-)
1. [x] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/tutorials/stateful-application/cassandra.md 1 file changed, 1 deletion(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `dev-1.16-ko.1` branch.

For example:
```
# checkout `dev-1.16-ko.1`
...
# check what is updated in the upstream 
git diff upstream/dev-1.15-ko.5 dev-1.16-ko.1 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `dev-1.16-ko.1`
```

### PR DESCRIPTION
**Pull request by gochist**
### 17 files to be modified 
1. [x] content/en/_index.html 1 file changed, 1 insertion(+)
1. [x] content/en/docs/concepts/architecture/cloud-controller.md 1 file changed, 5 insertions(+), 4 deletions(-)
1. [x] content/en/docs/concepts/architecture/nodes.md 1 file changed, 11 insertions(+)
1. [x] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 48 insertions(+), 4 deletions(-)
1. [x] content/en/docs/concepts/overview/what-is-kubernetes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 21 insertions(+), 2 deletions(-)
1. [x] content/en/docs/reference/_index.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/reference/glossary/applications.md 1 file changed, 1 deletion(-)
1. [x] content/en/docs/reference/glossary/static-pod.md 1 file changed, 8 insertions(+), 4 deletions(-)
1. [x] content/en/docs/reference/glossary/workload.md 1 file changed, 8 insertions(+), 14 deletions(-)
1. [x] content/en/docs/setup/_index.md 1 file changed, 2 insertions(+), 1 deletion(-)
1. [x] content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [x] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 4 insertions(+)
1. [x] content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md 1 file changed, 147 insertions(+), 100 deletions(-)
1. [x] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [x] content/en/docs/tutorials/stateful-application/cassandra.md 1 file changed, 1 deletion(-)

fixes #16438

### CODE REVIEW

File: content/ko/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -190,9 +190,20 @@ DaemonSet 컨트롤러에 의해 생성된 파드는 쿠버네티스 스케줄
...
+활성화 시켜두면, kubelet이 리소스 할당 결정을 할 때 토폴로지 힌트를 사용할 수 있다.
 
 ## API 오브젝트
 
 노드는 쿠버네티스 REST API 내 탑-레벨 리소스 이다. API 오브젝트에 대한 보다 자세한 내용은 [노드 API 오브젝트](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#node-v1-core)에서 확인할 수 있다.
 
 {{% /capture %}}
+{{% capture whatsnext %}}
+* [노드 컴포넌트](https://kubernetes.io/docs/concepts/overview/components/#node-components)에 대해 읽기
+* 노드 수준 토폴로지에 대해 읽기: [Control Topology Management Policies on a node](/docs/tasks/administer-cluster/topology-manager/)
```
Comment (seokho-son) [inline content/ko/docs/concepts/architecture/nodes.md]: Control Topology Management Policies on a node 부분 번역이 가능해 보입니다.

File: content/ko/docs/concepts/containers/runtime-class.md [hunk 2]
```diff
@@ -144,6 +143,42 @@ https://github.com/containerd/cri/blob/master/docs/config.md
 더 자세한 cri-o의 구성 문서를 살펴본다.
 https://github.com/kubernetes-sigs/cri-o/blob/master/cmd/crio/config.go
 
+### 스케줄
+
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
+
+쿠버네티스 v1.16 부터는, 런타임 클래스는 `scheduling` 필드를 통해 이종의 클러스터 지원을 포함한다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/containers/runtime-class.md]:
```suggestion
쿠버네티스 v1.16 부터, 런타임 클래스는 `scheduling` 필드를 통해 이종의 클러스터 지원을 포함한다.
```
조금 자연스럽게 변경해 보았습니다.

File: content/ko/docs/concepts/containers/runtime-class.md [hunk 3]
```diff
@@ -172,4 +207,11 @@ https://github.com/kubernetes-sigs/cri-o/blob/master/cmd/crio/config.go
   더 이상 올바르지 않으며, 반드시 올바른 핸들러 구성으로 이전헤야 한다
   (위를 참조).
 
+### 더 읽기
+
+- [RuntimeClass Design](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/runtime-class.md)
```
Comment (seokho-son) [inline content/ko/docs/concepts/containers/runtime-class.md]: 번역이 가능해 보이는 부분입니다.

File: content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 4]
```diff
@@ -304,10 +349,12 @@ AKS-Engine은 완전하고, 맞춤 설정이 가능한 쿠버네티스 클러스
 
 #### kubeadm과 클러스터 API로 배포하기
 
-Kubeadm은 쿠버네티스 클러스터를 배포하는 사용자에게 산업 표준이 되었다. Kubeadm에서 윈도우 노드 지원은 미래 릴리스에 나올 것이다. 또한 윈도우 노드가 올바르게 프로비저닝되도록 클러스터 API에 투자하고 있다.
+Kubeadm은 쿠버네티스 클러스터를 배포하는 사용자에게 산업 표준이 되었다. Kubeadm에서 윈도우 노드 지원은 쿠버네티스 v1.16 이후 부터 알파 기능이다. 또한 윈도우 노드가 올바르게 프로비저닝되도록 클러스터 API에 투자하고 있다. 보다 자세한 내용은, [kubeadm for Windows KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/kubeadm/20190424-kubeadm-for-windows.md)를 통해 상담하도록 하자.
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md]:
```suggestion
Kubeadm은 쿠버네티스 클러스터를 배포하는 사용자에게 산업 표준이 되었다. Kubeadm에서 윈도우 노드 지원은 쿠버네티스 v1.16 이후 부터 알파 기능이다. 또한 윈도우 노드가 올바르게 프로비저닝되도록 클러스터 API에 투자하고 있다. 보다 자세한 내용은, [Windows KEP를 위한 kubeadm](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/kubeadm/20190424-kubeadm-for-windows.md)을 통해 상담하도록 하자.
```

File: content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md [hunk 5]
```diff
@@ -167,132 +176,168 @@ v1.14 이후의 최신 바이너리를 [https://github.com/kubernetes/kubernetes
...
-| 파라미터 | 기본값 | 비고 |
-| --- | --- | --- |
-| -ManagementIP | N/A (필요함) | 윈도우 노드에 할당할 IP 주소. 이 값을 찾기 위해 `ipconfig` 이용할 수 있다. |
-| -NetworkMode | l2bridge | 여기서는 `overlay`를 사용한다. |
-| -ClusterCIDR | 10.244.0.0/16 | 클러스터 IP 설계을 참고한다. |
-| -ServiceCIDR | 10.96.0.0/12 | 클러스터 IP 설계을 참고한다. |
-| -KubeDnsServiceIP | 10.96.0.10 | |
-| -InterfaceName | Ethernet | 윈도우 호스트의 네트워크 인터페이스 이름. 이 값을 찾기 위해 <code>ipconfig</code> 이용할 수 있다. |
-| -LogDir | C:\k | Kubelet과 Kube-proxy가 각각의 로그 출력 파일을 리다이렉션하는 디렉터리이다. |
+1. 컨트롤 플레인("Master") 노드에 SSH로 접속해서 [Kubeconfig 파일](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)을 얻어온다.
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md]:
```suggestion
1. 컨트롤 플레인("마스터") 노드에 SSH로 접속해서 [Kubeconfig 파일](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)을 얻어온다.
```
문서 내에서 "마스터"나 "Master"로 일관성 유지가 필요해 보입니다.

**Comment by gochist** [comment]
@seokho-son 감사합니다! 의견 반영해서 업데이트 했습니다.

**Comment by seokho-son** [comment]
Thank you !!
/lgtm
/approve

**Pull request merged**